scripts/fork_my_feedstocks.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env conda-execute
+
+"""
+This script can be run to fork conda-forge feedstocks to which you are a maintainer.
+This is super useful if you maintain many feedstocks and would like to cutdown maintenance on your next PR...
+
+Requires a token stored in the environment variable `GH_TOKEN` with the permissions `public_repo` and `read:org`.
+It also requires all the feedstocks be cloned somewhere like with the `feedstocks` repo.
+"""
+# conda execute
+# env:
+#  - git
+#  - python
+#  - conda-smithy
+#  - pygithub
+#  - gitpython
+# channels:
+#  - conda-forge
+# run_with: python
+
+import os
+import argparse
+
+import git
+import github
+
+from conda_build.metadata import MetaData
+
+import conda_smithy.github
+import conda_smithy.configure_feedstock
+import conda_smithy
+import conda_smithy.feedstocks as feedstocks
+
+
+parser = argparse.ArgumentParser(description="Fork your maintained feedstocks.")
+parser.add_argument("--feedstocks-dir", help="The location of the feedstocks.",
+                    default="~/Developer/Conda/conda-forge/feedstocks/feedstocks")
```

Comment:
I tend to try and make this relative so that there is at least a chance that it can be a sensible default for some. Not critical though.